cypress-integration/support/commands.js

```diff
@@ -39,3 +39,51 @@ Cypress.Commands.add('goToRoute', (...args) => {
     w.__cy_route(...args);
   });
 });
+
+Cypress.Commands.add('waitUntilSettled', (maxTries = 20) => {
```

Comment:
This is cool!  You should give the rest of the team a quick demo on when we should and shouldn't use this new function.